creusot-contracts/src/std/ord.rs

```diff
@@ -0,0 +1,101 @@
+use crate as creusot_contracts;
+use creusot_contracts_proc::*;
+
+use std::cmp::Ordering;
+
+pub trait Ord {
+    #[logic]
+    fn cmp_log(self, _: Self) -> Ordering;
```

Comment:
We probably will want to separate this out into `OrdLogic` eventually so we can have orderings for purely logical types (`Int`).